docs/pages/reference/mesheryctl/mesheryctl-pattern-list.md

```diff
@@ -48,7 +49,7 @@ mesheryctl pattern list
 
 <pre class='codeblock-pre'>
 <div class='codeblock'>
-      --config string   path to config file (default "/home/runner/.meshery/config.yaml")
+      --config string   path to config file (default "/home/dusdjhyeon/.meshery/config.yaml")
```

Comment:
it is correct to point config file path `/home/dusdjhyeon/.meshery/config.yaml` instead of `/home/user/.meshery/config.yaml` like something.